src/style.css

```diff
@@ -1 +1,88 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: rgb(255 255 255);
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav_item {
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  padding: 0;
+  margin-left: 20px;
+  font-size: 12px;
+  box-sizing: border-box;
+
+  display: flex;
+  align-items: center;
+  position: relative;
```

Comment:
Please pay attention to your CSS properties order
https://9elements.com/css-rule-order/